<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CollectionCoinManagementDialog</class>
 <widget class="QDialog" name="CollectionCoinManagementDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>771</width>
    <height>596</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <widget class="QGroupBox" name="groupBoxManagement">
     <property name="title">
      <string>Collection Management</string>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLineEdit" name="lineEditCollectionName">
          <property name="placeholderText">
           <string>Collection Name</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QGroupBox" name="groupBoxCollectionContents">
          <property name="title">
           <string>Collection Contents</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_2">
             <item>
              <widget class="QLabel" name="label_3">
               <property name="text">
                <string/>
               </property>
               <property name="pixmap">
                <pixmap resource="qresource.qrc">:/ico/cash.ico</pixmap>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="label">
               <property name="text">
                <string>Available Coins From Server</string>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="horizontalSpacer_2">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <widget class="QLabel" name="label_2">
               <property name="text">
                <string>Coins You Want To Track</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="label_4">
               <property name="text">
                <string/>
               </property>
               <property name="pixmap">
                <pixmap resource="qresource.qrc">:/ico/cash.ico</pixmap>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QGridLayout" name="gridLayout">
             <item row="0" column="0" rowspan="5">
              <widget class="QListWidget" name="listWidgetAvailableCoins"/>
             </item>
             <item row="0" column="1">
              <widget class="QPushButton" name="pushButtonMoveItemToLeft">
               <property name="text">
                <string/>
               </property>
               <property name="icon">
                <iconset resource="qresource.qrc">
                 <normaloff>:/ico/leftArrow.ico</normaloff>:/ico/leftArrow.ico</iconset>
               </property>
              </widget>
             </item>
             <item row="0" column="2" rowspan="5">
              <widget class="QListWidget" name="listWidgetTrackedCoins"/>
             </item>
             <item row="1" column="1">
              <widget class="QPushButton" name="pushButtonMoveItemsToRight">
               <property name="text">
                <string/>
               </property>
               <property name="icon">
                <iconset resource="qresource.qrc">
                 <normaloff>:/ico/rightArrow.ico</normaloff>:/ico/rightArrow.ico</iconset>
               </property>
              </widget>
             </item>
             <item row="2" column="1">
              <widget class="QPushButton" name="pushButtonMoveAllToLeft">
               <property name="text">
                <string/>
               </property>
               <property name="icon">
                <iconset resource="qresource.qrc">
                 <normaloff>:/ico/doubleLeftArrow.ico</normaloff>:/ico/doubleLeftArrow.ico</iconset>
               </property>
              </widget>
             </item>
             <item row="3" column="1">
              <widget class="QPushButton" name="pushButtonMoveAllToRight">
               <property name="text">
                <string/>
               </property>
               <property name="icon">
                <iconset resource="qresource.qrc">
                 <normaloff>:/ico/doubleRightArrow.ico</normaloff>:/ico/doubleRightArrow.ico</iconset>
               </property>
              </widget>
             </item>
             <item row="4" column="1">
              <spacer name="verticalSpacer">
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>20</width>
                 <height>40</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonControllerBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="qresource.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonControllerBox</sender>
   <signal>rejected()</signal>
   <receiver>CollectionCoinManagementDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
